README.md

```diff
@@ -118,15 +125,69 @@ Exit Code Number   | Meaning
 `126`              | A service inside the box is not running / Command invoked cannot execute
 
 
-## IP Address Detection <a name="ip_addr"></a>
+<a name="ip-address-detection"></a>
+## IP address detection
 
 There is no standardized way of detecting Vagrant box IP addresses.
 This code uses the last IPv4 address available from the set of configured addresses that are *up*.  i.e. if eth0, eth1, and eth2 are all up and have IPv4 addresses, the address on eth2 is used.
 
 
-## Getting Involved with the Project <a name="Contributing"></a>
+<a name="development"></a>
+# Development
+
+<a name="setup"></a>
+## Setup
+
+After cloning the repository, install the [Bundler](http://bundler.io/) gem:
+
+    $ gem install bundler
+
+Then setup your project dependencies:
+
+    $ bundle install
+
+The build is driven via rake. All build related tash should be executed in the
+Bundler environment, e.g. `bundle exec rake clean`. You can get a list of available
+Rake tasks via:
+
+    $ bundle exec rake -T
+
+<a name="acceptance-tests"></a>
+## Acceptance tests
+
+The source contains also a set of [Cucumber](https://cucumber.io/) acceptance tests. They can be run via:
+
+    $ bundle exec rake features
+
+The tests assume that the ADB and CDK box files are available under
+_build/boxes/adb-\<provider\>.box_ resp _build/boxes/cdk-\<provider\>.box_. You can
+either copy the box files manually or use the _get_adb_ resp. _get_cdk_ Rake tasks.
+
```

Comment:
This will save some internet bandwidth for me as I dont have  to re-download box again which is already on the workstation i.e Fedora. I had downloaded the box using following commands
```
vagrant init projectatomic/adb
vagrant up
```
The Vagrant box is kept inside /home/lmohanty/.vagrant.d/boxes/projectatomic-VAGRANTSLASH-adb/2.0.0 extracted format i.e
```
[lmohanty@LalatenduM-laptop projectatomic-VAGRANTSLASH-adb]$ cd 2.0.0/
[lmohanty@LalatenduM-laptop 2.0.0]$ tree ./
./
└── libvirt
    ├── box.img
    ├── metadata.json
    └── Vagrantfile

1 directory, 3 files
```